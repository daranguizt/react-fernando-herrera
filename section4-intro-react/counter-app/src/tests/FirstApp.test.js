import React from 'react';
import {shallow} from 'enzyme';
const { render } = require("@testing-library/react");
const { default: FirstApp } = require("../FirstApp");

// describe('tests on <FirstApp />', () => {
//     test('should show "Hola Mundo, Diego"', () => {
//         const msg = 'Diego';
//         const {getByText} = render(<FirstApp msg={msg}/>);
        
//         expect(getByText(`Hola mundo, ${msg}`)).toBeInTheDocument();
//     });
// });

describe('test on <FirstApp />', () => {
    test('should show FirstApp', () => {
        const msg = 'Diego';
        const wrapper = shallow(<FirstApp msg={msg}/>);

        expect(wrapper).toMatchSnapshot();
    })
});