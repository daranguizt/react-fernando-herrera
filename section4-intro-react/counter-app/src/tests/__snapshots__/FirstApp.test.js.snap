// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test on <FirstApp /> should show FirstApp 1`] = `
<Fragment>
  <h1>
    Hola mundo
    , 
    Diego
  </h1>
  <h1>
    1
    2
    3
    4
    5
    6
  </h1>
  <h1>
    {"nombre":"Fernando","edad":25}
  </h1>
  <h2>
    Soy un subtítulo
  </h2>
  <p>
    Mi primer párrafo
  </p>
</Fragment>
`;
